source/client/ClickableImage.py

```diff
@@ -19,12 +19,8 @@ def __init__(self, image, x, y, width, height, outline_index):
         self.outline_color = UIC.outline_colors[outline_index]
 
     def draw(self, display, loc_xy, outline_choice):
-        """ Call this method to draw the ClickableImage on the screen,
-        
-        Future debug -- do I really need loc_xy in this?  Wrappedcard has a loc_xy already!
-        Could replace this with new routine called clickableCard that uses fact that Image
-        and location from same object.
-        """
+        # Call this method to draw the ClickableImage on the screen
```

Comment:
Why no longer a doc string? triple quoted comments after a method def are standard even if its just the one line.